datafusion-cli/src/exec.rs

```diff
@@ -250,37 +245,43 @@ pub(super) async fn exec_and_print(
             }
             // As the input stream comes, we can generate results.
             // However, memory safety is not guaranteed.
```

Comment:
not related to this PR, but I don't understand this comment about Memory safety

Reply:
It's also confusing to me,  maybe we can remove the message. It should be memory safe, we only keep one batch size.